src/ast/nodes/VariableDeclaration.ts

```diff
@@ -36,12 +36,10 @@ export default class VariableDeclaration extends NodeBase {
 	}
 
 	includeWithAllDeclaredVariables() {
-		let anotherPassNeeded = false;
 		this.included = true;
 		for (const declarator of this.declarations) {
-			if (declarator.include()) anotherPassNeeded = true;
```

Comment:
Interesting this is no longer needed.

Reply:
The dirty-flag is now managed via the context but it seems this file was overlooked. Nice that TypeScript found this!